iati/home/tests/test_middleware.py

```diff
@@ -0,0 +1,49 @@
+"""A module of unit tests for redirects."""
+import pytest
+from django.conf import settings
+from factories import DocumentFactory
+
+
+@pytest.mark.django_db
+class TestRedirectMiddleware():
+    """Tests for the custom redirect middleware."""
+
+    def create_document(self):
+        """Call the DocumentFactory to save a new document."""
+        return DocumentFactory.create()
+
+    @pytest.mark.parametrize('redirect_mapping', [
+        ('/HoMe/', '/home/'),
+        ('/HOME/', '/home/'),
+        ('/home/', '/en/home/'),
+        ('/104/MiXeDCaSe/', 'http://reference.iatistandard.org/104/MiXeDCaSe/'),
+        ('/105/MiXeDCaSe/', 'http://reference.iatistandard.org/105/MiXeDCaSe/'),
+        ('/201/MiXeDCaSe/', 'http://reference.iatistandard.org/201/MiXeDCaSe/'),
+        ('/202/MiXeDCaSe/', 'http://reference.iatistandard.org/202/MiXeDCaSe/'),
+        ('/203/MiXeDCaSe/', 'http://reference.iatistandard.org/203/MiXeDCaSe/'),
+        ('/activity-standard/MiXeDCaSe/', 'http://reference.iatistandard.org/activity-standard/MiXeDCaSe/'),
+        ('/codelists/MiXeDCaSe/', 'http://reference.iatistandard.org/codelists/MiXeDCaSe/'),
+        ('/developer/MiXeDCaSe/', 'http://reference.iatistandard.org/developer/MiXeDCaSe/'),
+        ('/introduction/MiXeDCaSe/', 'http://reference.iatistandard.org/introduction/MiXeDCaSe/'),
+        ('/namespaces-extensions/MiXeDCaSe/', 'http://reference.iatistandard.org/namespaces-extensions/MiXeDCaSe/'),
+        ('/organisation-identifiers/MiXeDCaSe/', 'http://reference.iatistandard.org/organisation-identifiers/MiXeDCaSe/'),
+        ('/organisation-standard/MiXeDCaSe/', 'http://reference.iatistandard.org/organisation-standard/MiXeDCaSe/'),
+        ('/reference/MiXeDCaSe/', 'http://reference.iatistandard.org/reference/MiXeDCaSe/'),
+        ('/rulesets/MiXeDCaSe/', 'http://reference.iatistandard.org/rulesets/MiXeDCaSe/'),
+        ('/schema/MiXeDCaSe/', 'http://reference.iatistandard.org/schema/MiXeDCaSe/'),
+        ('/upgrades/MiXeDCaSe/', 'http://reference.iatistandard.org/upgrades/MiXeDCaSe/'),
+        (settings.STATIC_URL + "HOME/css/", "/en" + settings.STATIC_URL + "HOME/css/")
```

Comment:
Use [string formatting](https://docs.python.org/3.4/library/stdtypes.html#str.format) instead of concatenation for strings